src/frontend/typer/søknad.ts

```diff
@@ -261,6 +261,7 @@ export const initialStateSøknad: ISøknad = {
             poststed: '',
         },
         utenlandsperioder: [],
+        arbeidsperioder: [],
```

Comment:
Kanskje vi kan droppe å sette noe på søknadsobjektet denne runden, og lage en egen PR for de 3 modalene vi har nå? Litt usikker på hvordan det vil bli med mellomlagring ogsånn

Reply:
Sant, dette hang igjen fra tidligere 😅